src/app/foundations/chart/chart-demo.component.ts

```diff
@@ -0,0 +1,194 @@
+import {Component} from '@angular/core';
+import {SectionService} from '../../shared/section.service';
+import {BaseDemoComponent} from '../../shared/base-demo.component';
+import { SelectChangeEvent } from '@healthcatalyst/cashmere';
+import {ViewChild, ElementRef} from '@angular/core';
+import * as echarts from 'echarts';
+import * as hcChartLight from '../../../assets/foundations/charts/healthcatalyst-light-echart-v5.x.json';
+import * as hcChartDark from '../../../assets/foundations/charts/healthcatalyst-dark-echart-v5.x.json';
```

Comment:
These JSON theme files are static and hard coded. In our react component library I set these chart themes up to be dynamically created based on the currently loaded theme. I was not sure of an easy way to provide that functionality from the docs. Maybe a charting component in the future would be a good candidate for dynamic chart themes.